doc-content/enterprise-only/openshift/operator-deploy-kieserver-proc.adoc

```diff
@@ -0,0 +1,103 @@
+** `h2`: A built-in `h2` database engine that does not require a separate pod. Do not scape the {KIE_SERVER} pod if you use this setting.
+** `external`: An external database server. If you use any database server except PostgreSQL or MySQL, you must provide a {KIE_SERVER} image with the database server driver according to instructions in <<externaldb-build-proc>>.
+. Optionally, in the *Size* field, enter the size of the peristence volume to create for the database server.
+. If you selected an external database server, complete the following additional steps:
+.. In the *Driver* field, enter the database server driver, depending on the server type:
++
+*** `mysql`
+*** `postgresql`
+*** `mariadb`
+*** `mssql`
+*** `db2`
+*** `oracle`
+*** `sybase`
++
+.. In the *Dialect* field, enter the Hibernate dialect for the server, depending on the server type:
++
+*** `org.hibernate.dialect.MySQL5InnoDBDialect` (used for MySQL and MariaDB)
+*** `org.hibernate.dialect.PostgreSQL82Dialect`
+*** `org.hibernate.dialect.SQLServer2012Dialect` (used for MS SQL)
+*** `org.hibernate.dialect.DB2Dialect`
+*** `org.hibernate.dialect.Oracle10gDialect`
+*** `org.hibernate.dialect.SybaseASE157Dialect`
++
+.. In the *Host* field, enter the host name of the external database server.
+.. In the *Port* field, enter the port number of the external database server.
+.. In the *Jdbc URL* field, enter the JDBC URL for the external database server.
+.. If you want to configure the data source in non-XA mode, select the *NonXA* box.
+.. In the *JNDI name* field, enter the JNDI name that the application uses for the data source.
+.. In the *User name* and *Password* fields, enter the user name and password for the external database server.
+.. Optionally, set the minimum and maximum connection pool sizes, valid connection checker class, and exception sorter class for the database server.
+.. Optionally select the *Background validation* box to enable background SQL valication and enter the background validation interval.
+endif::PAM[]
+. Optionally, depending on your needs, set environment variables. To set an environment variable, click *Add new Environment variable*, then enter the name and value for the variable in the *Name* and *Value* fields.
+** If you want to configure your {KIE_SERVER} deployment to use Prometheus to collect and store metrics, set the `PROMETHEUS_SERVER_EXT_DISABLED` environment variable to `false`. For instructions about configuring Prometheus metrics collection, see {URL_MANAGING_KIE_SERVER}#prometheus-monitoring-ocp-proc_execution-server[_{MANAGING_KIE_SERVER}_].
+** If you want to use an external Maven repository, set the following variables:
+*** `MAVEN_REPO_URL`: The URL for the Maven repository.
+*** `MAVEN_REPO_ID`: An identifier for the Maven repository, for example, `repo-custom`.
+*** `MAVEN_REPO_USERNAME`: The username for the Maven repository.
```

Comment:
"The user name for"  Also, remove the periods from all the bulletted lines in this list.